tests/unit/versioning/VersionID.spec.js

```diff
@@ -9,83 +10,124 @@ function randkey(length) {
     return key;
 }
 
-describe('test generating versionIds', () => {
-    const count = 1000;
+function generateRandomVIDs(count) {
     const vids = Array(count).fill(null);
     for (let i = 0; i < count; i++) {
         vids[i] = VID.generateVersionId(randkey(15), 'PARIS');
     }
     process.env.VID_CRYPTO_PASSWORD = randkey(64);
+    return vids;
+}
+
+const count = 1000000;
+
+describe('test generating versionIds', () => {
+    describe('legaxy hex encoding', () => {
+        env.S3_VERSION_ID_ENCODING_TYPE = 'hex';
+        const vids = generateRandomVIDs(count);
```

Comment:
Linked with my other comment, we may also move the vid generation outside of the encoding-specific test block, after `const count = 1000000`

Reply:
the env var `env.S3_VERSION_ID_ENCODING_TYPE = 'hex';` needs to be set before the run. Otherwise the info field gets chopped off by default.